frontend/src/Frontend/UI/Dialogs/Send.hs

```diff
@@ -527,13 +553,15 @@ finishCrossChainTransferConfig model fromAccount ucct = Workflow $ do
     dialogSectionHeading mempty "Gas Payer (recipient chain)"
     sender <- divClass "group" $ elClass "div" "segment segment_type_tertiary labeled-input" $ do
       divClass "label labeled-input__label" $ text "Account Name"
-      let cfg = def & dropdownConfig_attributes .~ pure ("class" =: "labeled-input__input")
+      let cfg = def & dropdownConfig_attributes .~ pure ("class" =: "labeled-input__input select select_mandatory_missing")
           chain = pure $ Just toChain
       gasAcc <- uiSenderDropdown cfg model chain never
       pure $ ffor3 (model ^. wallet_accounts) (model ^. network_selectedNetwork) gasAcc $ \netToAccount net ma -> do
-        accounts <- Map.lookup net $ unAccountStorage netToAccount
-        a <- ma
-        lookupAccountRef a accounts
+        accounts <- Map.lookup net $ unAccountData netToAccount
+        n <- ma
+        AccountInfo _ chains <- Map.lookup (fst n) accounts
+        _acc <- Map.lookup fromChain chains
```

Comment:
Sanity check or leftover?
Maybe use `guard`

Reply:
Sanity check. `guard` works for me.